packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginFormError.js

```diff
@@ -1,26 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Collapse from 'react-collapse';
-import { default as cx } from 'classnames';
 
-const LoginFormError = ({ children, show, className, ...props }) =>
-  children && (
-    <Collapse {...props} isOpened={show} className={cx('login-form-error', className)}>
+const LoginFormError = ({ children, show, topErrorOnly, ...props }) => (
+  <div className="login-form-error" style={topErrorOnly ? { minHeight: '20px' } : null}>
```

Comment:
can we add an additional class rather than use inline styling to allow applications to override this setting?

Reply:
Thanks, fixed that